pkg/cmd/cli/cmd/status.go

```diff
@@ -72,45 +97,56 @@ func RunStatus(f *clientcmd.Factory, out io.Writer) error {
 	if err != nil {
 		return err
 	}
+	o.namespace = namespace
 
-	describer := &describe.ProjectStatusDescriber{K: kclient, C: client, Server: config.Host}
-	s, err := describer.Describe(namespace, "")
-	if err != nil {
-		return err
-	}
+	o.describer = &describe.ProjectStatusDescriber{K: kclient, C: client, Server: config.Host, Suggest: o.suggest}
+
+	o.out = out
 
-	fmt.Fprintf(out, s)
 	return nil
 }
 
-// RunGraph contains all the necessary functionality for the OpenShift cli graph command
-func RunGraph(f *clientcmd.Factory, out io.Writer) error {
-	client, kclient, err := f.Clients()
-	if err != nil {
-		return err
+// Validate validates the options for the Openshift cli status command.
+func (o StatusOptions) Validate() error {
+	if o.hasArgs {
+		return errors.New("no arguments should be provided")
```

Comment:
Users won't know what ** means - so it's probably better to have something
like:

* you have blah blah
  fix: change blah to blah

On Mon, Nov 30, 2015 at 9:13 AM, Michail Kargakis <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/status.go
> <https://github.com/openshift/origin/pull/6078#discussion_r46146495>:
>
> >  	return nil
> >  }
> >
> > -// RunGraph contains all the necessary functionality for the OpenShift cli graph command
> > -func RunGraph(f *clientcmd.Factory, out io.Writer) error {
> > -	client, kclient, err := f.Clients()
> > -	if err != nil {
> > -		return err
> > +// Validate validates the options for the Openshift cli status command.
> > +func (o StatusOptions) Validate() error {
> > +	if o.hasArgs {
> > +		return errors.New("no arguments should be provided")
>
> ok
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6078/files#r46146495>.
>